### PR TITLE
docs: replace deprecated karma-coverage-istanbul-reporter

### PR DESCRIPTION
**Pull request by manueledones**
'karma-coverage-istanbul-reporter' usage has been deprecated since version 11.
Please install 'karma-coverage' and update 'karma.conf.js.' For more info, see https://github.com/karma-runner/karma-coverage/blob/master/README.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by AndrewKushnir** [comment]
@kyliau could you please help with the review of this change? Thank you.

File: aio/content/guide/testing-code-coverage.md [hunk 1]
```diff
@@ -38,19 +38,26 @@ The code coverage percentages let you estimate how much of your code is tested.
...
-  reports: [ 'html', 'lcovonly' ],
-  fixWebpackSourcePaths: true,
-  thresholds: {
-    statements: 80,
-    lines: 80,
-    branches: 80,
-    functions: 80
-  }
-}
+coverageReporter: {
```
Comment (kyliau) [inline aio/content/guide/testing-code-coverage.md:45]: Please fix indentation

**Review by kyliau — APPROVED** [review]

File: aio/content/guide/testing-code-coverage.md [hunk 2]
```diff
@@ -38,19 +38,26 @@ The code coverage percentages let you estimate how much of your code is tested.
 If your team decides on a set minimum amount to be unit tested, you can enforce this minimum with the Angular CLI.
 
 For example, suppose you want the code base to have a minimum of 80% code coverage.
-To enable this, open the [Karma](https://karma-runner.github.io) test platform configuration file, `karma.conf.js`, and add the following in the `coverageIstanbulReporter:` key.
+To enable this, open the [Karma](https://karma-runner.github.io) test platform configuration file, `karma.conf.js`, and add the `check` property in the `coverageReporter:` key.
+You can find the configuration [here](https://github.com/karma-runner/karma-coverage/blob/master/docs/configuration.md).
```
Comment (gkalpak) [inline aio/content/guide/testing-code-coverage.md:42]: I would put this sentence below the code snippet and change it to something like:

```suggestion
You can find more info about the different coverage configuration options [here](https://github.com/karma-runner/karma-coverage/blob/master/docs/configuration.md).
```

File: aio/content/guide/testing-code-coverage.md [hunk 3]
```diff
@@ -38,19 +38,26 @@ The code coverage percentages let you estimate how much of your code is tested.
...
 
 ```
```
Comment (gkalpak) [inline aio/content/guide/testing-code-coverage.md:44]:
```suggestion
```js
```

File: aio/content/guide/testing-code-coverage.md [hunk 4]
```diff
@@ -38,19 +38,26 @@ The code coverage percentages let you estimate how much of your code is tested.
...
-  fixWebpackSourcePaths: true,
-  thresholds: {
-    statements: 80,
-    lines: 80,
-    branches: 80,
-    functions: 80
-  }
-}
+coverageReporter: {
+      dir: require('path').join(__dirname, './coverage/...'),
```
Comment (gkalpak) [inline aio/content/guide/testing-code-coverage.md:46]: I think we should either remove the existing properties (and only show the newly added `check` property) or make it more clear that the path should include the project's name:

```suggestion
      dir: require('path').join(__dirname, './coverage/<project-name>'),
```

...or...

```suggestion
      dir: require('path').join(__dirname, './coverage/angular.io-example'),
```

(`angular.io-example` is the project name used in the associated StackBlitz live example.)

File: aio/content/guide/testing-code-coverage.md [hunk 5]
```diff
@@ -38,19 +38,26 @@ The code coverage percentages let you estimate how much of your code is tested.
...
+      ],
+      check: {
+        global: {
+          statements: 80,
+          branches: 80,
+          functions: 80,
+          lines: 80
+        }
+      }
+    }
```
Comment (gkalpak) [inline aio/content/guide/testing-code-coverage.md:60]: This snippet is not indented correctly.

**Review by AndrewKushnir — APPROVED** [review]
LGTM too once comments from Keen are addressed.

@manueledones thanks for improving Angular docs 👍

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch! Thx, @manueledones :+1:

I left a couple fo comments/suggestions. As you make the changes, could you also remove the ` (#1)` suffix from the commit message header? :pray:

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**